build/parser.js

```diff
@@ -465,13 +465,14 @@ class Parser {
    * Limit items to tradable/untradable if specified.
    */
   addTradable (item) {
-    const tradableTypes = ['Upgrades', 'Fish', 'Key', 'Focus Lens', 'Relic', 'Rifle Mod',
+    const tradableTypes = ['Arcane', 'Fish', 'Focus Lens', 'Relic', 'Rifle Mod',
       'Secondary Mod', 'Shotgun Mod', 'Warframe Mod', 'Companion Mod', 'Archwing Mod', 'K-Drive Mod',
       'Melee Mod']
-    const untradableTypes = ['Skin', 'Medallion', 'Extractor', 'Pets', 'Ship Decoration']
-    const tradableRegex = /(Prime|Vandal|Wraith|Rakta|Synoid|Sancti|Vaykor|Telos|Secura)/i
-    const untradableRegex = /(Glyph|Mandachord|Greater.*Lens|Sugatra)/i
-    const notFiltered = !untradableTypes.includes(item.type) && !item.name.match(untradableRegex)
+    const untradableTypes = ['Skin', 'Medallion', 'Key', 'Extractor', 'Pets', 'Ship Decoration',
+      'Glyph', 'Sigil', 'Fur Color', 'Syandana', 'Fur Pattern', 'Color Palette', 'Node', 'Exalted Weapon']
+    const tradableRegex = /(Prime|Vandal|Wraith|Rakta|Synoid|Sancti|Vaykor|Telos|Secura|Ayatan|Prisma)/i
+    const untradableRegex = /(Glyph|Mandachord|Greater.*Lens|Sugatra|\[|SentinelWeapons|Toroid|Bait|([A-Za-z]+ (Relic)))/i
+    const notFiltered = !untradableTypes.includes(item.type) && !item.name.match(untradableRegex) && !item.uniqueName.match(untradableRegex) && (item.hasOwnProperty('productCategory') ? !item.productCategory.match(/(SpecialItems)/) : true)
```

Comment:
could you wrap some of the lines down, like pull the options in the ternary so it's readable? i probably need to add an editorconfig with line length so this is enforced if it's not covered in the existing linting